services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function does not handle the case where a movie session with the specified ID does not exist. Consider handling the `MovieSession.DoesNotExist` exception to avoid potential errors.